Week3/README.md

```diff
@@ -10,26 +10,32 @@ In week three we will discuss the following topics:
 • Functions
 ```
 
+
+> :boom: IMPORTANT: The concepts of these weeks are extremely important, and require very careful reading. 
+> Make sure you understand these concepts as well as you can, as they are crucial to programming and your ability to do well in our program!
+
+
+
+
 ### Here are resources that we like you to read as a preparation for the coming lecture:
 
 Please watch the following parts of the course, [Programming Foundations Fundamentals](https://www.lynda.com/Programming-Foundations-tutorials/Welcome/83603/90426-4.html) on Lynda.com (if you don't have access to Lynda yet ask Gijs):
+
     <br>4. Writing Conditional Code
     <br>5. Modular Code
     <br>6. Iteration: Writing Loops
     <br>7. More About Strings
     <br>8. Collections
     <br>11. When Things Go Wrong 
```

Comment:
Can you get rid of the <br> tags? They show up verbatim as this is a code block.